packages/debezium-server.nix

```
@@ -19,18 +18,23 @@ in
       hash = "sha256-y7elU/9zBjGSpBFwAXHvhZvLb0QvvBazwDNP9yBQdJw=";
     };
 
-    nativeBuildInputs = [makeWrapper];
-
     installPhase = ''
       runHook preInstall
       cp -R . $out
-      RUNNER=$(ls $out/debezium-server-*runner.jar)
-      PATH_SEP=":"
+      mkdir -p $out/bin
+      cat >$out/bin/run_debezium <<EOF
+      #!$SHELL
+      RUNNER=\$(ls $out/debezium-server-*-runner.jar)
```

```suggestion
      RUNNER="$out/debezium-server-dist-${version}-runner.jar"
```

I saw there's usage of `debezium-server-*runner.jar` in the `run.bat/sh` scripts, but the above serves the same function in a more nix-y way. We then end up with the following lines at the top of the `run_debezium` script.

```
RUNNER="/nix/store/8vdds0y92g54j96lz05snczphjqbwzjw-debezium-server-2.7.1.Final/debezium-server-dist-2.7.1.Final-runner.jar"
LIB_PATH="/nix/store/8vdds0y92g54j96lz05snczphjqbwzjw-debezium-server-2.7.1.Final/lib/*"
```

---

Good idea!.. made this and ran a nix build locally to confirm working okay.